evm/forge-test/TokenBridgeRelayer.t.sol

```diff
@@ -1298,65 +1425,25 @@ contract TokenBridgeRelayerTest is Helpers, ForgeHelpers, Test {
             toNativeTokenAmount,
             normAmounts.tokenDecimals
         );
-
-        // test setup
-        {
-            // target contract setup
-            avaxRelayer.registerToken(avaxRelayer.chainId(), wrappedAsset);
-
-            // register this contract as the foreign emitter
-            avaxRelayer.registerContract(
+        normAmounts.relayerFee = normalizeAmount(
+            avaxRelayer.calculateRelayerFee(
                 ethereumChainId,
-                addressToBytes32(address(this))
-            );
-
-            // set the native swap rate
-            avaxRelayer.updateSwapRate(
-                avaxRelayer.chainId(),
                 wrappedAsset,
-                6.9e2 * avaxRelayer.swapRatePrecision() // swap rate
-            );
-
-            // set the max to native amount
-            avaxRelayer.updateMaxNativeSwapAmount(
-                avaxRelayer.chainId(),
-                wrappedAsset,
-                6.9e18 // max native swap amount
-            );
-
-            /**
-             * NOTE: The relayer fee is calculated on the source chain, and the
-             * target relayer contract will pay the relayer the encoded value. We
-             * need to simulate calculating the value based on information stored
-             * in the target chain.
-             */
-
-            // set relayer fee to 20 USD
-            avaxRelayer.updateRelayerFee(
-                ethereumChainId,
-                20 * avaxRelayer.relayerFeePrecision()
-            );
-
-            normAmounts.relayerFee = normalizeAmount(
-                avaxRelayer.calculateRelayerFee(
-                    ethereumChainId,
-                    wrappedAsset,
-                    normAmounts.tokenDecimals
-                ),
                 normAmounts.tokenDecimals
-            );
+            ),
+            normAmounts.tokenDecimals
+        );
 
-            // make some assumptions about the fuzz test values
-            vm.assume(
-                normAmounts.transferAmount > 0 &&
-                amount < type(uint96).max
-            );
-            vm.assume(
-                normAmounts.toNative > 0 &&
-                toNativeTokenAmount < type(uint96).max &&
-                normAmounts.transferAmount > normAmounts.toNative + normAmounts.relayerFee
-            );
-        }
+        // make some assumptions about the fuzz test values
+        vm.assume(
+            normAmounts.transferAmount > 0 &&
+            amount < type(uint96).max
+        );
+        vm.assume(
+            normAmounts.toNative > 0 &&
+            toNativeTokenAmount < type(uint96).max &&
+            normAmounts.transferAmount > normAmounts.toNative + normAmounts.relayerFee
+        );
```

Comment:
[vm.assume()](https://book.getfoundry.sh/cheatcodes/assume) actually reverts the test. You might consider constraining these numbers using [bound](https://book.getfoundry.sh/reference/forge-std/bound) where possible from forge-std as it limits the value mathematically instead of completely throwing away that full test run like vm.assume does.

An example might look like:
```solidity
toNativeTokenAmount = bound(toNativeTokenAmount, 0, type(uint96).max - 1)
```